programs/client/BuzzHouse/Generator/ExternalIntegrations.h

```diff
@@ -0,0 +1,953 @@
+#pragma once
+
+#include "config.h"
+
+#include "FuzzConfig.h"
+#include "SQLCatalog.h"
+
+#if defined USE_MYSQL && USE_MYSQL
+#    if __has_include(<mysql.h>)
+#        include <mysql.h>
+#    else
+#        include <mysql/mysql.h>
+#    endif
+#endif
+
+#if defined USE_MONGODB && USE_MONGODB
+#    include <bsoncxx/builder/stream/array.hpp>
+#    include <bsoncxx/builder/stream/document.hpp>
+#    include <bsoncxx/json.hpp>
+#    include <bsoncxx/types.hpp>
+#    include <mongocxx/client.hpp>
+#    include <mongocxx/collection.hpp>
+#    include <mongocxx/database.hpp>
+#endif
+
+#if defined USE_LIBPQXX && USE_LIBPQXX
+#    include <pqxx/pqxx>
+#endif
+
+#if defined USE_SQLITE && USE_SQLITE
+#    include <sqlite3.h>
+#endif
+
+
+namespace BuzzHouse
+{
+
+using IntegrationCall = enum IntegrationCall { IntMySQL = 1, IntPostgreSQL = 2, IntSQLite = 3, IntRedis = 4, IntMongoDB = 5, IntMinIO = 6 };
```

Comment:
We should be using an `enum class` instead. Among others, that'd allow us to remove the `Int` prefix of all names, since they won't clash.

```suggestion
using IntegrationCall = enum class IntegrationCall { MySQL = 1, PostgreSQL = 2, SQLite = 3, Redis = 4, MongoDB = 5, MinIO = 6 };
```

Do not accept the suggestion as it is, because you'll need to manually refactor all references in other files.

Reply:
MySQL, PostgeSQL and SQLite names already exist in BuzzHouse namespace, they need something different here.

Reply:
AFAIK, if you use an `enum class` you can call them internally however you want